src/ParachainRegistry.sol

```diff
@@ -13,8 +13,24 @@ interface IRegistry {
         bytes palletInstance;
         uint256 weightToFee;
         XcmTransactorV2.Multilocation feeLocation;
+        uint64 reportStakeDeposited;
```

Comment:
Why not just use the weights struct here? e.g. `Weights weights` if that works?

Reply:
yes it'll work, thanks for the suggestion.